modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,23 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Checklist 2.0.5
+
+The {productname} 6.5 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.5 includes one fix:
+
+=== Checklist toggling was possible on `contenteditable="false"` elements
+//#TINY-9469
+
+In earlier versions of {productname}, the Checklist plugin allowed toggling on elements with the `contenteditable = "false"` attribute.
+
+As a consequence, lists nested within a `contenteditable = "false"` section of the editor could be converted into a checklist.
+
+To address this issue, the Checklist button has now been disabled for `contenteditable = "false"` sections in the {productname} 6.5.
+
+As a result, it is no longer possible to apply the Checklist functionality to `contenteditable = "false"` sections.
+
```

Comment:
```suggestion
=== Checklist 2.0.5

The {productname} 6.5 release includes an accompanying release of the **Checklist** premium plugin.

**Checklist** 2.0.5 includes one fix:

=== Checklist toggling was possible on elements with a `contenteditable="false"` attribute set
//#TINY-9469

In previous versions of {productname}, the Checklist plugin could toggle elements with a `contenteditable = "false"` attribute set.

As a consequence, lists nested within an element with a `contenteditable="false"` attribute set could be converted into checklists.

To address this, as of {productname} 6.5, the Checklist button is disabled when the selection is within an element with a `contenteditable="false"` attribute set.

It is no longer possible to apply Checklist functionality to elements with a `contenteditable="false"` attribute set.

For information on the **Checklist** plugin, see: xref:checklist.adoc[Checklist].
```

1. Copy-edits.
2. Long-and-formal term copied in to place.
3. _For information_ line added.